### PR TITLE
pp-31 feat: add 5 types of button component

### PR DESCRIPTION
**Pull request by nadiakohli**

### CODE REVIEW

File: src/components/common/buttons/IconButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  color = 'primary',
+  onClick = () => { console.log('click'); },
+}: IButtonProps) => (
+  <ButtonStyle
+    type={type}
+    color={color}
+    disabled={disabled}
+    onClick={onClick}
+  >
+    {Icon && <Icon />}
```
Comment (nadiakohli) [inline src/components/common/buttons/IconButton/index.tsx:28]:
```suggestion
    {Icon ? <Icon /> : null}
```

File: src/components/common/buttons/IconTextButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  color = 'primary',
+  onClick = () => { console.log('click'); },
+}: IButtonProps) => (
+  <ButtonStyle
+    type={type}
+    color={color}
+    disabled={disabled}
+    onClick={onClick}
+  >
+    {Icon && <Icon />}
```
Comment (Voy19) [inline src/components/common/buttons/IconTextButton/index.tsx:30]: Add conditional

File: src/components/common/buttons/IconTextButton/styled.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+// Constants
+import { COLORS } from '@constants/index';
```
Comment (Voy19) [inline src/components/common/buttons/IconTextButton/styled.ts:4]: delete index

File: src/components/common/buttons/TagButton/index.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+const TagButton: FC<IButtonProps> = ({
+  Icon,
+  children,
+  disabled = false,
+  color = 'primary',
+  onClick = () => { console.log('click'); },
+}: IButtonProps) => (
+  <ButtonStyle
```
Comment (Voy19) [inline src/components/common/buttons/TagButton/index.tsx:22]: the same about one line

**Review by Voy19 — APPROVED** [review]

**Pull request merged**